server/payment.go

```diff
@@ -0,0 +1,49 @@
+package server
+
+import (
+	"github.com/bitcoin-sv/go-paymail"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+func (c *Configuration) GetPaymailAndCreateMetadata(context *gin.Context, satoshis uint64) (alias, domain string, md *RequestMetadata, ok bool) {
```

Comment:
We should add a comment for this exported method.

Reply:
Added